src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -30,6 +28,12 @@ object DplaMapData {
   */
 case class DplaMapData(
                          sourceResource: DplaSourceResource,
+                         /*
```

Comment:
This is based on the MAP spec. EdmWebResource here describes the file format and the rights. There may be URI redundancy however.